test/implement_pitch_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class TestPitchImplementation < MiniTest::Test
+  def setup
+    @pitch_note_B4      = MusicTheory::Note.new(pitch: Pitch.new(:B, 4), duration: 1.0, distort: false)
```

Comment:
In order of how loosely coupled they'll be:
1. `Note` doesn't know anything about `Pitch`:
```ruby
MusicTheory::Note.new(frequency: Pitch.new(:B, 4).frequency)
```
2. `Note` knows how to ask `Pitch` for `frequency`: 
```ruby
MusicTheory::Note.new(pitch: Pitch.new(:B, 4)
```
3. `Note` knows how to construct `Pitch` from a specially formatted raw data structure and ask it for `frequency`:
```ruby
MusicTheory::Note.new(pitch: [:B, 4])
```